lib/services/access_request.go

```diff
@@ -46,12 +46,12 @@ func ValidateAccessRequest(ar types.AccessRequest) error {
 
 // NewAccessRequest assembles an AccessRequest resource.
 func NewAccessRequest(user string, roles ...string) (types.AccessRequest, error) {
-	return NewAccessRequestWithResources(user, roles, nil)
+	return NewAccessRequestWithResources(user, roles, []types.ResourceID{})
```

Comment:
Can we make the code work with `nil` slice instead of empty slice?

Reply:
it does, I'm just being explicit here, I don't like seeing `nil` and having it not be completely obvious what it is